src/main/java/ch/njol/skript/entity/SimpleEntityData.java

```diff
@@ -212,6 +232,33 @@ private static void addSuperEntity(String codeName, Class<? extends Entity> enti
 
 		if (Skript.isRunningMinecraft(1,21,3)) {
```

Comment:
```suggestion
		if (Skript.isRunningMinecraft(1,21,2)) {
```
same thing here, as a safety precaution